### ISSUE TITLE
Change storage location

### ISSUE THREAD

**Issue by maxgoedjen** [kitematic/kitematic#247]
Putting a "Kitematic" folder in the users home directory is considered to be super gross on OS X. the proper location for those files is `~/Application Support/Kitematic`

**Comment by dfinninger** [comment]
On my setup it is, `~/Library/Application Support/Kitematic`

**Comment by maxgoedjen** [comment]
![](http://f.cl.ly/items/2i2i3V3A3p0W3i3N3m3W/Image%202015-03-12%20at%201.28.36%20PM.png) is what I'm seeing here (installed via the download link on the homepage today, 0.5.10)

**Comment by dfinninger** [comment]
@maxgoedjen Yes, that is where is is for me as well currently. I meant to clear this up with my edit. On my computer "Application Support" is located under `~/Library`, so when putting the Kitematic stuff on my system it *should* be located at `~/Library/Application Support/Kitematic`.

**Comment by maxgoedjen** [comment]
Ah yes, my mistake. Updating the issue to reflect it.

**Comment by jeffdm** [comment]
App data (like the sqlite database for `localStorage`) is in `~/Library/Application Support/Kitematic`.

That said, only container volume data is placed in the created directory `~/Kitematic` for easy access to volume data. Where do you feel makes the most sense for host-mounted container volume data?

**Comment by maxgoedjen** [comment]
To be honest app support seems like the best place to me still (especially given how easy it is to bring up the folder from the Kitematic UI). `~/Documents` seems like the next best, ideally you'd be able to set that directory in settings.

**Comment by jeffdm** [comment]
Good suggestion. Thanks!

**Comment by TeckniX** [comment]
Somehow I don't have the Kitematic folder created on mine - The 'hello-world-nginx' disk image and rethinkdb have been installed, yet I don't see where the files are saved.

**Comment by flowirin** [comment]
i've just downloaded a 3gb container. can't find it anywhere (no, its not in ~/Library/Application Support/Kitematic)

where does 0.7.4 store downloaded VMs?

**Comment by maxgoedjen** [comment]
@flowirin this ticket is surrounding the container volume (kind of like a virtual hard drive for the container), not the actual image for the container itself.

**Comment by MuTLY** [comment]
Using Windows here, and the Kitematic folder is created inside my "Documents" folder. This is the default, but I don't think it should be there.

Example: I have my Documents folder actually pointing to OneDrive. So every time I create something in there, it gets uploaded to OneDrive.

This is fine for Word, Excel, some other stuff, but it gets ugly when using Kitematic.

Isn't there a way to configure Kitematic to NOT use the "default" folder? E.g., I want to use my D: drive and a folder in its root directory, like D:\Kitematic.

**Comment by FrenchBen** [comment]
The 'My Documents' folder is safely shared with all laptop installs of Docker and why it's the default. You can use a different folder, as long as you remember to enable sharing for that folder.

**Comment by petyunchik** [comment]
Could it be an option at least?
